### DISCUSSION TITLE
Expose main.j2.tex?

### DISCUSSION THREAD

**Discussion by rdeoliveira** [rendercv/rendercv#274] [Q&A] [answered]
I'm looking to create a custom theme that changes the entire **layout** of the doc, in addition to its contents and style.

I followed these instructions and ran `rendercv create-theme`, but it seems that the `.j2.tex` files it creates only govern the contents of the doc, not the layout. 

It seems that to customise the layout, I'd need to customise [rendercv/themes/main.j2.tex](https://github.com/rendercv/rendercv/rendercv/themes/main.j2.tex), but it is not exposed to us with `rendercv create-theme`.

So should `main.j2.tex` be exposed? Or is there another way to customise the entire layout of the doc?

PS: Thanks for this amazing tool!

**Comment by sinaatalay** [comment, marked as answer]
You're welcome! `main.j2.tex` can be overridden by having it next to the YAML input file. It's not documented, but it works.

**Reply by rdeoliveira** [reply]
Indeed it worked!

Although for PDF only:

<img width="1440" alt="image" src="https://github.com/user-attachments/assets/70ebf5ab-2985-412e-958f-4cda4036d799" />

Any way to generate HTML/MD also with the custom `main.j2.tex`?
